### ISSUE TITLE
Cant wait to try MICo-150K

### ISSUE THREAD

**Issue by vvvvvjdy** [A113N-W3I/MICo-150K#1]
Nice work！ Will the dataset opensourced in the coming days？

**Comment by A113N-W3I** [comment]
For sure! We are in the process of open-sourcing everything. Due to network restrictions on our servers, uploading to Hugging Face is taking longer than expected. However, we will release the complete dataset as well as all fine-tuned model checkpoints.

**Comment by aihao2000** [comment]
@A113N-W3I Request to share the training parameters of QwenImage-MICo, such as batch size and learning rate.
